### PR TITLE
Improved the information displayed in the banner when administering OS Profiles

### PR DESCRIPTION
**Pull request by nroUSS38**
release Note updated

### CODE REVIEW

File: admin/en/source/pages/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,15 @@
 Changelog
 =========
 
+3.7.fp6
+-------
+Release Date: 2017-07-25
```
Comment (yanngit) [inline admin/en/source/pages/changelog.rst]: Please keep a blank line before Release like done before

File: admin/en/source/pages/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,15 @@
...
+
+Enhancements
+~~~~~~~~~~~~
+* Improved the information displayed in the banner when administering OS Profiles. Now the date the OS profile was created is displayed (for Windows), as well as the size and the associated distribution (for both Windows and Linux)
```
Comment (yanngit) [inline admin/en/source/pages/changelog.rst]: Creation date (for Windows only), size and associated distribution are now displayed in the OS Profile banner in administration page.

File: admin/en/source/pages/changelog.rst (hunk 1 above)
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: agree with Yann

File: admin/en/source/pages/changelog.rst (hunk 2 above)
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: original wording ok for me, but ok to change (for Windows only) instead of (for Windows).

**Pull request merged**